pkg_defs/ska3-flight/meta.yaml

```diff
@@ -11,31 +11,35 @@ requirements:
   run:
     - ska3-core ==2020.01.10
     - ska3-template ==2019.09.03
-    - acdc ==4.4.1
-    - agasc ==4.8.0
-    - acisfp_check ==2.7.2
+    - aca_view ==0.3.0
+    - acdc ==4.5.0
+    - agasc ==4.8.1
+    - acisfp_check ==3.0.0
     - acis_taco ==4.1.0
-    - acis_thermal_check ==2.9.2
-    - annie ==0.9.1
-    - backstop_history ==1.1.2
-    - chandra_aca ==4.28.1
+    - acis_thermal_check ==3.0.0
+    - annie ==0.10.1
+    - backstop_history ==1.1.3
+    - bep_pcb_check ==3.0.0
     - chandra.cmd_states ==3.15.1
     - chandra.maneuver ==3.7.2
     - chandra.time ==3.20.4
+    - chandra_aca ==4.29.1
     - cxotime ==3.1.1
-    - dea_check ==2.3.2
-    - dpa_check ==2.5.2
+    - dea_check ==3.0.0
+    - dpa_check ==3.0.0
     - find_attitude ==3.3.2
+    - fep1_actel_check ==3.0.0
+    - fep1_mong_check ==3.0.0
     - hopper ==4.4.1
     - jplephem ==2.8
     - kadi ==5.0.1
     - maude ==3.3.1
     - mica ==4.20.0
```

Comment:
Should update mica to 4.21.0

Reply:
For this kind of update in the PR when ska3-flight 2020.04.24 may already exist but *not* have mica, I've generally just incremented the metapackage manually (say to 2020.04.25) but looking forward to better ideas/implementations!